### ISSUE TITLE
[1.7.10] FM Shards  and Custom NetherDimensions

### ISSUE THREAD

**Issue by Grambosch** [SpitefulFox/ForbiddenMagic#103]
Hi there! We are using Forbidden Magic in Combination with  a custom generated Nether (FWG)
and we werent able to get any FM Shards from Mobs so far. So our Question ist , is your Mod Checking for Dim -1 ? And if so how can we work around this?

**Comment by SpitefulFox** [comment]
Yes, it's specifically checking for Dim -1.  No idea how you'd work around it other than adding some custom recipes.
